tensorboard/webapp/runs/views/runs_table/runs_data_table.ts

```diff
@@ -36,32 +36,39 @@ export class RunsDataTable {
   @Input() headers!: ColumnHeader[];
   @Input() data!: TableData[];
   @Input() sortingInfo!: SortingInfo;
+  @Input() experimentIds!: string[];
 
   ColumnHeaderType = ColumnHeaderType;
 
   @Output() sortDataBy = new EventEmitter<SortingInfo>();
   @Output() orderColumns = new EventEmitter<ColumnHeader[]>();
   @Output() onSelectionToggle = new EventEmitter<string>();
   @Output() onAllSelectionToggle = new EventEmitter<string[]>();
+  @Output() onRunColorChange = new EventEmitter<{
+    runId: string;
+    newColor: string;
+  }>();
+
+  beforeColumns = [
+    {
+      name: 'selected',
+      displayName: '',
+      type: ColumnHeaderType.CUSTOM,
+      enabled: true,
+    },
+  ];
+
+  afterColumns = [
+    {
+      name: 'color',
+      displayName: '',
+      type: ColumnHeaderType.COLOR,
+      enabled: true,
+    },
+  ];
 
   getHeaders() {
-    return [
-      {
-        name: 'selected',
-        displayName: '',
-        type: ColumnHeaderType.CUSTOM,
-        enabled: true,
-      },
-    ].concat(
-      this.headers.concat([
-        {
-          name: 'color',
-          displayName: '',
-          type: ColumnHeaderType.COLOR,
-          enabled: true,
-        },
-      ])
-    );
+    return this.beforeColumns.concat(this.headers.concat(this.afterColumns));
```

Comment:
nit: For some reason I find this to be hard to read.
```suggestion
    return this.beforeColumns.concat(this.headers, this.afterColumns);
```
or
```suggestion
    return [].concat(this.beforeColumns, this.headers, this.afterColumns);
```